sld/records.nycmesh.net.tf

```diff
@@ -201,7 +201,7 @@ resource "namedotcom_record" "record_nycmesh-375p-dns1-resolver_5233305" {
   answer      = "199.167.59.10"
 }
 
-# NS record for the mesh.nycmesh.net zone
+# NS record for the authoritative server for mesh.nycmesh.net at SN1
 resource "namedotcom_record" "record_mesh_5226462" {
```

Comment:
How will it work having more than one `NS` record for `mesh.nycmesh.net`? Should we remove this old one at the same time to make the behavior more predicable?

Reply:
It is not recommended to have only one authoritative DNS server, even if it is in a data center. In our case, the risk is higher because of nature of the (current) path between SN1 and the internet.
I'm hoping for:
1. SN1 (current)
2. SN1 + SN3
3. SN1 + SN3 + SN1{0,1}
4. SN3 + SN10 + SN11